automation_tools/satellite6/log.py

```diff
@@ -123,38 +137,15 @@ def _save_full_log(host, log_file_path, content):
         _print_wrapper('## Full upgrade logs saved on %s' % file_path)
 
 
-def analyze(host, log_file, content):
-    """Analyzes appended content from a log file. For now it is only
-    checking for ERROR status on regular log files.
-
-    :param host: str with host name
-    :param log_file: str log file path on host
-    :param content: str with log file content
-    """
-
-    def print_lines(lines_enumeration):
-        """print lines with numbers"""
-        no_line_flag = True
-        for i, line in lines_enumeration:
-            _print_wrapper('{}: {}'.format(i, line))
-            no_line_flag = False
-        if no_line_flag:
-            _print_wrapper('No errors found')
-
-    _print_wrapper('### Analyzing %s:' % log_file)
-    _print_wrapper('## Errors found:')
-    content_lines = content.decode('utf-8').split('\n')
-    regex = '|'.join(map(lambda token: '.*%s.*' % token, ERROR_TOKENS))
-    error_re = re.compile(regex, re.IGNORECASE)
-
-    lines_with_error_enum = filter(
-        lambda tpl: error_re.match(tpl[1]),
-        enumerate(content_lines, start=1))
-
-    print_lines(lines_with_error_enum)
-    _save_full_log(host, log_file, content)
-
-
 def _print_wrapper(s):
     """Just a wrapper to make mocking easier on tests"""
-    print(s.encode('utf-8'))
+    if isinstance(s, unicode):
+        print(s.encode('utf-8'))
+    else:
+        print(s)
+
+
+if __name__ == '__main__':
+    with LogAnalyzer('root@cloud-qe-15.idmqe.lab.eng.bos.redhat.com',
```

Comment:
Hard-coded Value ?

Reply:
I've forgot to remove the code a used to test against a real host. Problem fixed.